src/content.js

```diff
@@ -211,20 +221,18 @@ function handlePrCreatePage() {
                 async: false,
                 success: function(result) {
                     $('input#pull_request_title').val('['+ticketNumber.toUpperCase()+'] ' + result.fields.summary);
-                    /*ticketDescription = result.fields.description
-                        .replace('h1.', '# ')
-                        .replace('h2.', '## ')
-                        .replace('h3.', '### ')
-                        .replace('h4.', '#### ')
-                        .replace('h5.', '##### ')
-                        .replace('h6.', '###### ')
-                        .replace('{code}', '```' + NL)
-                        .replace('{{', '``')
-                        .replace('}}', '``');*/
+
+                    description = result.fields.description;
```

Comment:
Best case scenario - it works - but this _can_ be ``null``. So, please add a test for if that value is actually a string (or cast it to a string).

Other than that it works great and I think this is a good addition!

Once this is merged I'll bump the version and release a new one to the chrome webstore. Thanks for your effort!